avocado_virt/test.py

```diff
@@ -42,7 +42,7 @@ def _restore_guest_images(self):
                            compressed_drive_file)
             cwd = os.getcwd()
             os.chdir(os.path.dirname(compressed_drive_file))
-            process.run('xz -d %s' %
+            process.run('xz -d -k -f %s' %
```

Comment:
Please use the extended option (`--keep`, `--force`) for increased readability.While on it, please use `avocado.utils.path.find_command()` to probe the `xz` binary.

Reply:
And still while on it, s/`-d`/`--decompress`.

Reply:
OK, addressed the long args suggestion but I didn't add the "find_command" as it's a dependency and in case it's missing the "process" exception tells it all.

Reply:
No more dependencies that what we already have and with the benefit of not relying on `$PATH` only (we've been there[1]).
Please give it a second thought. If you insist, I will accept without it as it's not related to your PR.

[1] - https://www.redhat.com/archives/avocado-devel/2017-November/msg00017.html

Reply:
The user can disable "restore_image" or delete the "xz" file to avoid this dependency. I don't think we should make this a soft-fail.

Reply:
We are discussing different things. I have never requested to make it soft.  `find_comand()` raises an exception on 'command not found' as well. All I requested was to don't rely only on `$PATH`. Anyway, you seem pretty convinced about the best solution and this discussion is not really related to your PR, so I'm merging it. Thank you.